src/components/RecordDetail/UserDefined.astro

```diff
@@ -0,0 +1,29 @@
+---
+import { getTranslations } from "@i18n/server";
+import UserDefinedFieldView from "@components/UserDefinedFieldView";
+import config from "@config";
+
+const { record } = Astro.props
+
+const lang = Astro.currentLocale;
+const { t } = await getTranslations(lang);
+---
+{record.user_defined && (
+  <>
+    {Object.keys(record.user_defined).map(uuid => (
+      <>
+        {config.detail_pages?.exclude_fields && !config.detail_pages?.exclude_fields.includes(uuid) && (
```

Comment:
The exclude fields should also be able to suppress related records and native fields. See the implementation in the detail side panels.

Is there any reason we wouldn't want to use the same config option for the side panels and detail pages?

Reply:
> Is there any reason we wouldn't want to use the same config option for the side panels and detail pages?

Mixed feelings about this. If the user wants to filter the same set of fields it would be inconvenient to have to enter the data twice. But I can imagine them wanting to filter a different set of fields in each place.

Reply:
Let's discuss this in our meeting today.